RHDEVDOCS-6267 (#92126)
openshift/openshift-docs: configuring/configuring-openshift-builds.adoc

toc::[]

> In a `Build` custom resource (CR), you can define the source, build strategy,

Reviewer: ```suggestion
In a `Build` custom resource (CR), you can configure a build by defining the source, build strategy, parameter values, output, retention parameters, and volumes. You can also configure build pods by using a `Build` CR.
```
Similar edits here as what I suggested in `configuring-build-runs.adoc`. See my comments there.

> parameter values, output, retention parameters, and volumes to configure a build.

Reviewer: ```suggestion

```

> You can also use `Build` CR to configure build pods.

Reviewer: ```suggestion

```
